src/git.rs

```diff
@@ -239,4 +242,22 @@ mod tests {
         // Assert
         assert_eq!(expected_sections, actual_sections);
     }
+    #[test]
+    fn test_diff_several_extensions_files() {
```

Comment:
Really happy to see we have a new test !
As a followup maybe we could add an example with a .rs file and a non .rs file in the same diff ?

Reply:
I already add a test with a `.rs` and a `.toml` file, I think we can improve the test by adding a multi diff on the same `.rs` file for covering it all.

Reply:
Yeah I actually meant that.
We can do it in a followup PR though, it's pretty fine already :)